docs/enhancements/metallb-operator.md

```diff
@@ -0,0 +1,205 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@janedoe"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+---
+
+# Add MetalLB operator
+
+## Summary
+
+This enhancement focuses on the creation of relocatable clusters. A relocatable cluster is a cluster
+that can be moved from one location to another without cluster reconfigurations and without
+depending on a specific network topology. To this end, this enhancement proposaes adding support for
+the MetalLB operator to assisted installer.
+
+
+## Motivation
+
+The number of requests to create relocatable OCP clusters seem to be increasing and the idea of
+creating OCP appliances is becoming a common goal. Today there are solutions like ZTPFW that solve
+this problem by following a set of tasks that allow for these clusters to be relocated and for OCP
+to act as an appliance.
+
+Some of this tasks could be made part of components that are closer to the core so that they can be
+leveraged in other environments like SaaS. This enhancement proposes moving one of these tasks to
+Assisted Installer. Specifically, this enhancement proposes to deploy the MetalLB operator using
+Assisted Installer so that it can create clusters that can be relocated.
+
+### Goals
+
+- To add a MetalLB operator to Assiste Installer
```

Comment:
I have a draft PR here (not complete since it's waiting on the discussion in this enhancement): 

https://github.com/openshift/assisted-service/pull/4792